composer.json

```diff
@@ -43,7 +43,7 @@
     },
     "scripts": {
         "autoload": [
-            "echo '' > src/bootstrap.php",
+            "@php -r \"file_put_contents('src/bootstrap.php', '', LOCK_EX);\"",
```

Comment:
Maybe it's better to perform this logic in the build/bootstrap.php file instead then?
I think the idea behind this line was to clean it, so that the script fails it would be visible that the file was not generated. (not sure anymore :))

Reply:
Indeed. Error handling is missing from https://github.com/veewee/xml/blob/master/build/bootstrap.php

Reply:
True, it's a quick script to make sure everything is autoloaded. 
Don't think it makes sense to overengineer that

Reply:
Basically it boils down to `git diff --exit-code src/bootstrap.php`
That GHA workflow checks whether `src/bootstrap.php` changes after running the generator script.

A deletion command could be safe only **outside** `src/bootstrap.php`.